### PR TITLE
openmvs: bump version to latest release and fix build

### PR DESCRIPTION
**Pull request by TimPut**
###### Motivation for this change
openmvs-unstable was based on an old version of openmvs and has been broken for a long time.

###### Things done
This commit bumps openmvs to a stable release, bumps the CGAL dependency to a needed newer version and fixes the build.

https://github.com/cdcseacave/openMVS/issues/364#issuecomment-499842586

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/science/misc/openmvs/default.nix [hunk 1]
```diff
@@ -57,8 +56,7 @@ stdenv.mkDerivation {
     homepage = "http://cdcseacave.github.io/openMVS/";
     license = licenses.agpl3;
     platforms = platforms.linux;
-    maintainers = with maintainers; [ mdaiter ];
+    maintainers = with maintainers; [ mdaiter timput ];
     # 20190414-174115: CMake cannot find CGAL which is passed as build input
```
Comment (ajs124) [inline pkgs/applications/science/misc/openmvs/default.nix:60]: Is the comment still relevant if this is not broken anymore?

Also, I can verify that this builds (which I did), but I don't really know what this does, so I can't really test it.

Seeing as you're adding yourself as maintainer, I'd trust you that this actually does what it's supposed to.

Comment (TimPut) [reply pkgs/applications/science/misc/openmvs/default.nix]: You are correct about that comment, I've removed it now.

I've also changed the commit message to better align with CONTRIBUTING.md

**Comment by 7c6f434c** [comment]
@ofborg eval

**Comment by symphorien** [comment]
Please fix evaluation
```
undefined variable 'cgal_5' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-2/pkgs/applications/science/misc/openmvs/default.nix:14:45
```

**Comment by wegank** [comment]
Closed by #211780.

**Pull request closed without merging**